Description: Temporary kfreebsd bootstrap issue
 In fastcgi and fcl-web, something goes wrong with linking. Originally it was
   ELF interpreter /usr/libexec/ld-elf.so.1 not found
 which is fixed, but now it crashes when it is used (symbols issue I think) 
 .
 For now, I decide to remove fastcgi and fcl-web from the kfreebsd build.
 .
 Because of removing fastcgi and fcl-web from building, also fppkg fails:
    pkgfphttp.pp(18,12) Fatal: Can't find unit fphttpclient used by pkgfphttp
 This last failure can be prevented by building fcl-web until failure,
 then enough pieces are in place for fppkg to build. But as I haven't figured
 out yet how to do that sensibly, let's not build fppkg for now on kfreebsd.
Author: Paul Gevers <elbrus@debian.org>

--- fpc-2.6.2.orig/fpcsrc/packages/Makefile.fpc
+++ fpc-2.6.2/fpcsrc/packages/Makefile.fpc
@@ -4,7 +4,7 @@
 #
 
 [target]
-dirs=hash pasjpeg paszlib fpmkunit fcl-xml fcl-base fcl-db fcl-image fcl-net fcl-passrc fcl-registry fcl-fpcunit fcl-json fcl-js fcl-process unzip regexpr chm fcl-res libgd symbolic bzip2 hermes fppkg 
+dirs=hash pasjpeg paszlib fpmkunit fcl-xml fcl-base fcl-db fcl-image fcl-net fcl-passrc fcl-registry fcl-fpcunit fcl-json fcl-js fcl-process unzip regexpr chm fcl-res libgd symbolic bzip2 hermes 
 dirs_linux_i386=libc unixutil graph pxlib bzip2 
 dirs_i386_linux=graph
 dirs_x86_64_linux=graph
@@ -21,7 +21,7 @@ dirs_haiku=fv fcl-web fastcgi fcl-async
                gdbint libpng x11 gdbm tcl syslog libcurl opengl cairo bfd aspell svgalib \
                imlib utmp  fpgtk xforms fftw pcap ggi sdl openssl graph gnome1 gtk1 gtk2 librsvg  httpd22 httpd24  pxlib numlib \
                iconvenc dblib fpindexer
-dirs_freebsd=fv fcl-web fastcgi fcl-async ibase mysql ncurses zlib oracle odbc postgres sqlite pthreads imagemagick \
+dirs_freebsd=fv fcl-async ibase mysql ncurses zlib oracle odbc postgres sqlite pthreads imagemagick \
                gdbint libpng x11 gdbm tcl syslog libcurl opengl cairo  bfd aspell svgalib \
                imlib utmp  fpgtk xforms fftw pcap ggi sdl openssl graph gnome1 gtk1 gtk2 librsvg  httpd22 httpd24  pxlib numlib \
                users iconvenc gmp fcl-extra libxml dblib fpindexer libvlc
@@ -48,7 +48,7 @@ dirs_linux=fv fcl-web fastcgi fcl-async
                gdbint libpng x11 uuid ldap modplug dts mad  gdbm tcl syslog libcurl opengl opencl cairo gtk1 gtk2 librsvg \
                a52 bfd aspell svgalib newt cdrom users  iconvenc libxml proj4 fcl-extra zorba \
                imlib utmp  fpgtk openal lua oggvorbis xforms fftw pcap ggi sdl openssl gnome1  httpd22 httpd24  \
-               pxlib numlib gmp libsee ptc dblib fpindexer libvlc
+               pxlib numlib gmp libsee ptc dblib fpindexer libvlc fppkg
 dirs_win32=fv winunits-base winunits-jedi fcl-web fastcgi ibase mysql zlib oracle odbc postgres sqlite imagemagick \
                gdbint libpng mad tcl opengl gtk1 gtk2 librsvg a52 cdrom fpgtk openal fftw lua fcl-extra zorba \
                oggvorbis sdl openssl graph pcap  httpd22 httpd24  pxlib numlib winceunits cairo libxml \
--- fpc-2.6.2.orig/fpcsrc/utils/Makefile.fpc
+++ fpc-2.6.2/fpcsrc/utils/Makefile.fpc
@@ -7,7 +7,7 @@ name=utils
 version=2.6.5
 
 [target]
-dirs=fppkg fpcm tply h2pas fprcp dxegen fpdoc fpcmkcfg pas2ut pas2fpm
+dirs=fpcm tply h2pas fprcp dxegen fpdoc fpcmkcfg pas2ut pas2fpm
 programs=ppdep ptop rstconv data2inc delp bin2obj postw32 rmcvsdir relpath
 programs_linux=grab_vcsa
 dirs_win32=fpmc fpcres rmwait instantfpc importtl
@@ -14,7 +14,7 @@ dirs_win32=fpmc fpcres rmwait instantfpc
 dirs_win64=fpmc fpcres rmwait instantfpc importtl
 dirs_wince=fpcres rmwait instantfpc
 dirs_haiku=fpcres instantfpc
-dirs_linux=fpcres instantfpc
+dirs_linux=fpcres fppkg instantfpc
 dirs_freebsd=fpcres instantfpc
 dirs_openbsd=fpcres instantfpc
 dirs_netbsd=fpcres instantfpc
